### DISCUSSION TITLE
Share implementation of the milkdown playground

### DISCUSSION THREAD

**Discussion by zgrybus** [Milkdown/milkdown#915] [Q&A] [answered]
Hello,
May you share the implementation of the milkdown playground?
https://milkdown.dev/playground

I have no idea how to render toolbox :( 

Additionally, I have problem with https://stackblitz.com/github/Milkdown/examples/tree/main/react-block?file=README.md 
Popup never show, so it does not work for me :/

**Comment by Saul-Mirone** [comment, marked as answer]
1. https://github.com/Milkdown/website
2. The example doesn't have a popup.

**Comment by zgrybus** [comment]
Wonderful! Thanks :)
